css/main.css

```diff
@@ -28,3 +28,13 @@ body {
 
   text-align: right;
 }
+#mute{
```

Comment:
@zseiman can you move this to `utility.css`? This is to preserve the files' semantics. I think the mute button is a "utility" and shouldn't belong in `main.css`. Thanks.